core/src/components/datetime/test/basic/datetime.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from '@playwright/test';
+import type { E2EPage } from '@utils/test/playwright';
+import { test } from '@utils/test/playwright';
+
+test.describe('datetime: selecting a day', () => {
+  const testHighlight = async (page: E2EPage, datetimeID: string) => {
+    const today = new Date();
+    await page.goto('/src/components/datetime/test/basic');
+
+    const todayBtn = page.locator(
+      `#${datetimeID} .calendar-day[data-day='${today.getDate()}'][data-month='${today.getMonth() + 1}']`
+    );
+
+    const classes = await todayBtn.evaluate((el: any) => el.classList.value);
+    expect(classes).toMatch('calendar-day-today');
+    expect(classes).not.toMatch('calendar-day-active');
+
+    await todayBtn.click();
+    await page.waitForChanges();
+
+    const classes2 = await todayBtn.evaluate((el: any) => el.classList.value);
```

Comment:
You should be able to use the `toHaveClass` method instead: https://playwright.dev/docs/test-assertions#locator-assertions-to-have-class

Reply:
Same thing on line 14